src/SprykerEco/Zed/Unzer/Business/UnzerFacadeInterface.php

```diff
@@ -403,4 +404,25 @@ public function expandQuoteWithUnzerCredentials(QuoteTransfer $quoteTransfer): Q
      * @return \Generated\Shared\Transfer\UnzerPaymentTransfer|null
      */
     public function findUpdatedUnzerPaymentForOrder(OrderTransfer $orderTransfer): ?UnzerPaymentTransfer;
+
+    /**
+     * Specification:
+     * - Requires `UnzerMarketplacePaymentCredentialsResolverCriteriaTransfer.quote` transfer property to be set.
+     * - Requires `UnzerMarketplacePaymentCredentialsResolverCriteriaTransfer.paymentMethodKey` transfer property to be set.
+     * - Gets main marketplace Unzer credentials from persistence.
+     * - Performs Unzer get payment methods API call.
+     * - Returns payment method related Unzer credentials.
+     * - Throws `UnzerException` if Unzer Credentials for current store configuration could not be found in Persistence.
```

Comment:
use `@link` and full path to exception instead

Reply:
Fixed.